packages/patterns/src/constants.js

```diff
@@ -31,6 +31,7 @@ export const PARTIAL_SYNCING_SUPPORTED_BLOCKS = {
 		rel: __( 'Link Relationship' ),
 	},
 	'core/image': {
+		id: __( 'Image ID' ),
```

Comment:
There is a note in the code:

```js
// TODO: This should not be hardcoded. Maybe there should be a config and/or an UI.
```

@talldan and @SantosGuillamot, I contemplated the other day whether we should add labels to block attributes schema and annotate them as translatable in `block.json`. That could also be considered as one of the conditions to opt into block bindings in the future.

Reply:
By the way, I think you should use `_x()` with the context so it's easier to translate these labels.

https://github.com/WordPress/gutenberg/blob/a04a8e94e8b93ba60441c6534e21f4c3c26ff1bc/packages/blocks/src/api/registration.js#L271-L274

The context could be similar to:

https://github.com/WordPress/gutenberg/blob/a04a8e94e8b93ba60441c6534e21f4c3c26ff1bc/packages/blocks/src/api/i18n-block.json#L7

`block attribute label` could work.

Reply:
Yeah, I always have some doubts about including the translation strings here. Currently, they are not used in any way but are still kept. Should we just remove them for this release to not cause confusion for translators?

Reply:
Let's remove them to prevent unnecessary work for translators. 16 most popular locales need to have 100% coverage before the major release starts 😅